### DISCUSSION TITLE
golang-pkgs-for-production-server/

### DISCUSSION THREAD

**Discussion by giscus** [JungWinter/blog#44] [Comments]
# golang-pkgs-for-production-server/

전 직장에서 go 언어를 처음 쓰기 시작해 현 직장에서도 계속 쓰며 약 5년간 활발히 사용하고 있다.
예전과 비교하면 go 생태계는 점점 풍부해지고, 고퍼콘 코리아를 비롯한 다양한 커뮤니티도 늘어나고, 레퍼런스도 많아지고 있다.
그럼에도 여전히 (비…

https://www.winterjung.dev/golang-pkgs-for-production-server/

**Comment by Kcrong** [comment]
잘 읽었습니다. 감사합니다
